src/configuring-guacamole.md

```diff
@@ -750,6 +754,12 @@ only be accessed via VNC, one of these is likely your best choice. Both
 optimize window movement and (depending on the application) scrolling, giving a
 very responsive user experience.
 
+On Ubuntu Linux version 18.04, the package
+`tigervnc-standalone-server` does not support clipboard (copy/paste)
+functionality.  The `tigervncserver` packages provided by the TigerVNC
+project (https://sourceforge.net/projects/tigervnc/files/stable/) do
+support clipboard operations.
+
```

Comment:
My thought, here, is that, instead of trying to cover what the capabilities of specific distributions, versions, and VNC servers support, it might be better go more generic. Something more like `Your distribution and VNC server may not support clipboard functionality - please check the documentation or locate a VNC server/version that does.`

I'm just not sure the Guacamole Manual is the place to try to document (and keep up-to-date) the capabilities of various distributions and VNC servers. I don't think that'll scale well :-).

Reply:
Yes, I hear you.  But there is not other documentation to check, and how is one supposed to "find one that does".  I don't know what causes a VNC server to have clipboard functionality that works or doesn't with guacamole, but it's certainly not something they document in a clear way!  Would a "known good" / "known missing some functionality" list be more palatable?  It would at least be less wordy.

Reply:
I think I'd be okay with a couple of lists like that - these VNC servers are known to work, these are not, etc.